components/PostsComponent.jsx

```diff
@@ -39,20 +38,31 @@ const YearComponent = ({ year, posts }) => {
             <p className="text-2xl underline underline-offset-8 decoration-red-500 dark:decoration-red-400">
                 {year}
             </p>
-            {posts.map((post, idx) => (
-                <SinglePostComponent
-                    key={post.timestamp}
-                    post={post}
-                    isLastPost={idx === posts.length - 1}
-                />
+            {Object.entries(posts).map(([type, typePosts], idx) => (
```

Comment:
typePosts are bit misleading. Any better naming we can have?

Reply:
types or maybe typeArray?